org.sil.Bloom.yml

```diff
@@ -0,0 +1,647 @@
+              # Add bloom-join-team mime type
+              install -m 644 debian/JoinBloomTC.png ${INSTALLATION_PREFIX}/share/icons/hicolor/48x48/mimetypes/application-bloom-join-team.png
+              install -m 644 debian/JoinBloomTC.svg ${INSTALLATION_PREFIX}/share/icons/hicolor/scalable/mimetypes/application-bloom-join-team.svg
+              # Add AppStream metadata file
+              install -d ${INSTALLATION_PREFIX}/share/metainfo
+              install -m 644 ../local-extra/org.sil.Bloom.metainfo.xml ${INSTALLATION_PREFIX}/share/metainfo
+              # Remove unwanted files
+              cd ${BLOOMLIB} && rm -rf AxInterop* BloomTests* Interop* Moq* nunit* NUnit* AndikaNewBasic ffmpeg ghostscript
+              find ${BLOOMLIB} -name '*.map' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name '*Spec.js' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'CheckOrFixXliff.*' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'GeckofxHtmlToPdf.*' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name '*UnitTestFramework.dll' -type d -print0 | xargs -0 rm -rf
+              find ${BLOOMLIB} -name 'Moq.*' -print0 | xargs -0 rm -f
+              # We need NAudio.dll for SIL.Media to load even though it's Windows-only.  But we don't need the xml file.
+              find ${BLOOMLIB} -name 'NAudio.xml' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'irrKlang.NET4.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'win-x86' -type d -print0 | xargs -0 rm -rf
+              find ${BLOOMLIB} -name 'win-x64' -type d -print0 | xargs -0 rm -rf
+              find ${BLOOMLIB} -name '*Interop.AcroPDFLib.*' -print0 | xargs -0 rm -f
+              # These are needed to compile libpalaso on Windows, but aren't really used even there I suspect.
+              find ${BLOOMLIB} -name 'AtkSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'CairoSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'GdkSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'GioSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'GLibSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'GtkSharp.dll' -print0 | xargs -0 rm -f
+              find ${BLOOMLIB} -name 'PangoSharp.dll' -print0 | xargs -0 rm -f
+              # make up for deficient mono mapping
+              ln -s /usr/lib/x86_64-linux-gnu/libdl.so.2 ${INSTALLATION_PREFIX}/lib/libdl.so
+              # make room for the ffmpeg-full extension
+              install -d ${INSTALLATION_PREFIX}/lib/ffmpeg
+      cleanup:
+          - "/include"
+          - "/lib/*/include"
+          - "*.a"
+      sources:
+          - type: git
+            dest: BloomDesktop
+            url: https://github.com/BloomBooks/BloomDesktop.git
+            branch: Version5.2
```

Comment:
you need a commit or a tag, not a branch. NEVER a branch.

Reply:
changed to use the commit.